functions/system.bash

```diff
@@ -172,47 +165,49 @@ permissions_corrections() {
 
   if is_pine64; then
     cond_redirect groupadd gpio
-    cond_redirect cp $BASEDIR/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+    cond_redirect cp "$BASEDIR"/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+    # shellcheck disable=SC2016
     cond_redirect sed -i -e '$i \chown -R root:gpio /sys/class/gpio \n' /etc/rc.local
+    # shellcheck disable=SC2016
     cond_redirect sed -i -e '$i \chmod -R ug+rw /sys/class/gpio \n' /etc/rc.local
   fi
 
   for pGroup in audio bluetooth dialout gpio tty
   do
     if getent group "$pGroup" > /dev/null 2>&1 ; then
       cond_redirect adduser openhab "$pGroup"
-      cond_redirect adduser "$username" "$pGroup"
+      cond_redirect adduser "$username $pGroup"
```

Comment:
Why have you concatenated these variable, wouldn't this expose one varible to the `adduser` command?

Reply:
no but was unnecessary so changed it back

Reply:
Okey, my bad then. But I do prefer to have two `" ... "` blocks as it emphasizes that it is two variables.